platform-api/src/main/java/io/datacater/core/deployment/DeploymentConfig.java

```diff
@@ -0,0 +1,9 @@
+package io.datacater.core.deployment;
+
+public record DeploymentConfig(
```

Comment:
This looks like Streams-related configuration. Why is it replicated here for deployments?

Reply:
Thanks for pointing this out. I had made it to give the stream config to k8 class. Don't actually use it anymore, deleted.